nemo/collections/nlp/data/token_classification/punctuation_capitalization_dataset.py

```diff
@@ -142,6 +142,17 @@ class PunctuationCapitalizationDataConfigBase:
     tar_shuffle_n: int = 1
     """The size of shuffle buffer of `webdataset`. The number of batches which are permuted."""
 
+    shard_strategy: Optional[str] = 'scatter'
+    """Tarred dataset shard distribution strategy chosen as a str value during ddp. Accepted values are `scatter` and `replicate`.
+    `scatter`: The default shard strategy applied by WebDataset, where each node gets a unique set of shards, which are permanently
+    pre-allocated and never changed at runtime. `replicate` is an optional shard strategy, where each node gets all of the set of shards
+    available in the tarred dataset, which are permanently pre-allocated and never changed at runtime. The benefit of replication is that
+    it allows each node to sample data points from the entire dataset independently of other nodes, and reduces dependence on value of
+    `tar_shuffle_n`. Note: Replicated strategy allows every node to sample the entire set of available tarfiles, and therefore more than
```

Comment:
Single backquote is for italic in sphinx. In this case, perforated font suits more.
```suggestion
    ``tar_shuffle_n``. Note: Replicated strategy allows every node to sample the entire set of available tarfiles, and therefore more than
```

Reply:
Done. I've also updated docs.